aiida/cmdline/commands/cmd_devel.py

```diff
@@ -97,12 +98,16 @@ def devel_validate_plugins():
 
 @verdi_devel.command('run-sql')
 @click.argument('sql', type=str)
+@options.PROFILE()
```

Comment:
I think I added it here, because using the "global" profile meant it was being loaded and would fail if the storage schema was not at the head version, whereas with the code now, it is directly using `create_sqlalchemy_engine(profile)` and bypasses everything else, so you can run sql on old databases without migrating.
Will double check though

Reply:
If that mechanism is true, then I find that worrisome, because the `options.PROFILE` is available to anyone and so anyone could use it to access a backend without a schema check. I think this ties in to my other comment asking where the schema validation on backend load went. This is why in the current code the schema validation was _always_ performed, except in some rares cases that we used in our internal API where it was necessary to access an unmigrated backend. I really think we should keep this concept.

Reply:
The schema check is always performed, within `Backend.__init__`, but this is in `verdi devel` for a reason, it literally bypasses everything, if you look at the code:

```python
    with create_sqlalchemy_engine(profile).connect() as connection:
        result = connection.execute(text(sql)).fetchall()
```

Just accessing the database directly, outside of any aiida logic

Reply:
removed the option; it was failing because I still had `@decorators.with_dbenv()` there, which was previously required but not now

Reply:
I still see the option though. Should probably remove it

Reply:
its definitely gone